### DISCUSSION TITLE
osu!catch Lazer accuracy is unexpectedly high compared to stable

### DISCUSSION THREAD

**Discussion by SecreOsu** [ppy/osu#21846] [Q&A]
Currently lazer accuracy for catch is being calculated differently + way too high compared to stable. After testing stable and setting a singular SS score on a 30 second map, my profile accuracy jumped from 99.93 to 99.99
![before](https://user-images.githubusercontent.com/80791374/209722104-4c1082d6-fea6-4946-90b0-70500236ffab.png)
![after](https://user-images.githubusercontent.com/80791374/209722107-da23db34-93ea-4468-a230-aa79a0c54b24.png)

Either catch's lazer accuracy calculation got changed (and is horribly done) or there is something happening for a major discrepancy between stable/lazers calculation, both of which should be pretty large problems

**Comment by K3VRAL** [comment]
Could be in reference to #20508
